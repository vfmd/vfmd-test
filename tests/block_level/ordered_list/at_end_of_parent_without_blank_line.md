
### At the end of a sub-list

+ Parent list
  1. One
  2. Two
  3. Three
 
  4. Four
+ Parent list

### At the end of the document

1. One
2. Two
3. Three

4. Four
