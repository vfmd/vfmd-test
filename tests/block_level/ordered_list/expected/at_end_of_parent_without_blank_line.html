<h3>At the end of a sub-list</h3>
<ul>
  <li>
    Parent list

    <ol>
      <li>One</li>
      <li>Two</li>
      <li>Three</li>
      <li><p>Four</p></li>
    </ol>
  </li>
  <li>Parent list</li>
</ul>
<h3>At the end of the document</h3>
<ol>
  <li>One</li>
  <li>Two</li>
  <li>Three</li>
  <li><p>Four</p></li>
</ol>
